src/Entity/User.php

```diff
@@ -450,14 +450,34 @@ public function sendNotification(Notification $notification) {
         }
     }
 
+    private function _unreadNotificationsCriteria(Criteria $existingCriteria = null):Criteria 
+    {
```

Comment:
Hate to be that guy, but you're using both newline and sameline brackets :P

Reply:
I got used to K&R at FB, before I really liked Allman, I guess I will stick to whatever here it is